workflows/cloudify_system_workflows/snapshots/postgres.py

```diff
@@ -259,13 +260,37 @@ def dump_config_tables(self, tempdir):
         run_shell(command)
         return path
 
+    def dump_manager_tables(self, tempdir):
+        pg_dump_bin = os.path.join(self._bin_dir, 'pg_dump')
+        path = os.path.join(tempdir, 'manager_tables.dump')
+        command = [pg_dump_bin,
+                   '-a',
+                   '--host', self._host,
+                   '--port', self._port,
+                   '-U', self._username,
+                   self._db_name,
+                   '-f', path]
+
+        for table in self._MANAGER_TABLES:
+            command += ['-t', table]
+
+        run_shell(command)
+        return path
+
     def restore_config_tables(self, config_path):
         new_dump_file = self._prepend_dump(config_path, [
             'delete from {0};'.format(table)
             for table in self._CONFIG_TABLES
         ])
         self._restore_dump(new_dump_file, self._db_name)
 
+    def restore_manager_tables(self, manager_tables_path):
+        new_dump_file = self._prepend_dump(manager_tables_path, [
+            'delete from {0};'.format(table)
```

Comment:
Don't delete, it'll cascade- and we're only doing this when the tables will not be there anyway